File: lab-3-dynamodb/dynamoDBJavaLab/test/InfectionsDataUploaderTest.java
// Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights reserved.

import static org.junit.Assert.*;

import org.junit.Test;

public class InfectionsDataUploaderTest {

  @Test
  public void test() throws Exception {
    InfectionsDataUploader.main(new String[0]);
    assertEquals(1000, InfectionsDataUploader.numItemsAdded, 0);
  }
}

File: lab-3-dynamodb/dynamoDBPythonLab/infections_data_uploader_test.py
# Copyright 2017 Amazon Web Services, Inc. or its affiliates. All rights
# reserved.

import unittest
import infections_data_uploader
import warnings


class InfectionsDataUploaderTest(unittest.TestCase):

    def test_infections_uploader(self):
        warnings.simplefilter("ignore", ResourceWarning)
        numFailures = infections_data_uploader.load_infections_data()
        self.assertEqual(0, numFailures)

if __name__ == '__main__':
    unittest.main()
